### DISCUSSION TITLE
algebra relazionale - 5

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#62] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/17f775dd-034e-4265-8870-316975b62e59)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-07 130847](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/70319810-9400-40fa-b599-7fba4ebccf3c)
![Immagine 2024-02-07 130859](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/93ce2952-63ad-4633-90e4-3b2fd4079e40)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/ce557894-84b3-498e-b27a-a6786fd7dd41)
